code/index.html

```diff
@@ -2,29 +2,101 @@
+        <i class="fab fa-facebook-square"></i>
+        <i class="fab fa-twitter-square"></i></div>
+      <div class="header-title"><h1><a href="https://www.youtube.com/watch?v=fvk4PpFORD4" target="_blank">Ski Tripz</a></h1></div>
+      <div class="inspo"><p>Inspo up here!<span style='font-size:12px;'>&#10548;</span></p></div>
+      <div class="header-text"><h2>WIN an awesome skiing adventure!</h2></div>
+    </header>
+ 
+    <!-- The class 'parent' is the parent to classes 'form-container' and 'accordion'-->
+    <section class="parent">
+      <div class="form-container">
+        <h2>Sign up to participate</h2><br>
+          <form action="https://httpbin.org/anything" method="post" target="_blank">
+            <label for="firstname">First Name:</label><br>
+            <input type="text" id="firstname" name="firstname" placeholder="Enter your first name" required><br>
+            <label for="lastname">Last Name:</label><br>
+            <input type="text" id="lastname" name="lastname" placeholder="Enter your last name" required><br>
 
-    <!-- Signup form -->
+            <label for="email">Email:</label><br>
+            <input type="email" id="email" name="email" placeholder="Enter your e-mail" required ><br><br>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+            <p>Level of Experience:</p>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+            <input type="radio" id="beginner" name="experience-level" value="beginner" required>
+            <label class="experience-options" for="beginner">Beginner</label></br>
+            <input type="radio" id="intermediate" name="experience-level" value="intermediate" required>
+            <label class="experience-options" for="intermediate">Intermediate</label><br>
+            <input type="radio" id="advanced" name="experience-level" value="advanced" required>
+            <label class="experience-options" for="advanced">Advanced</label><br><br>
+
+            <p>Why should YOU win?</p><br>
+
+            <textarea name="message" rows="10" cols="30" placeholder="Describe your desire to win this awesome experience!" required></textarea><br><br>
+
+            <input type="checkbox" id="policy" name="policy" value="accepted" required>
+            <label class="checkboxes" for="policy"><i>Yes, I Accept all Terms & Conditions</i></label><br>
+            <input type="checkbox" id="newsletter" name="newsletter" value="yes">
+            <label class="checkboxes" for="newsletter"><i>Yes, I would like to receive your Newsletter</i></label><br><br>
+
+            <input class="submit-btn" type="submit" value="SUBMIT">
+          </form>
       </div>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <div class="accordion">
+        <h2>FAQ</h2>
+          <div class="contentBx">
```

Comment:
I think the guidelines are html/css = kebab-case, and JavaScript = camelCase, right? Not sure though

Reply:
Oh, that's right! Thank you for the input!